client/driver/rkt.go

```diff
@@ -21,8 +21,8 @@ import (
 )
 
 var (
-	reRktVersion  = regexp.MustCompile("rkt version ([\\d\\.]+).+")
-	reAppcVersion = regexp.MustCompile("appc version ([\\d\\.]+).+")
+	reRktVersion  = regexp.MustCompile("rkt version (\\d[.\\d]+)")
+	reAppcVersion = regexp.MustCompile("appc version (\\d[.\\d]+)")
```

Comment:
Can you change the it from quotes to backticks. That way you don't have to double delimit the backslashes.